stage1_fly/aci/aci-manifest.in

```diff
@@ -0,0 +1,33 @@
+{
+    "acKind": "ImageManifest",
+    "acVersion": "0.7.3",
+    "name": "coreos.com/rkt/stage1-fly",
+    "labels": [
+        {
+            "name": "version",
+            "value": "@RKT_STAGE1_VERSION@"
+        },
+        {
+            "name": "arch",
+            "value": "amd64"
+        },
+        {
+            "name": "os",
+            "value": "linux"
+        }
+    ],
+    "annotations": [
+        {
+            "name": "coreos.com/rkt/stage1/run",
+            "value": "/run"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/enter",
+            "value": "/enter"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/gc",
+            "value": "/gc"
```

Comment:
Same thing here: what about using value="" and testing that in stage0. For GC, we would just skip the stage1's GC if it is "". That would be less code to maintain.